docs/api.rst

```diff
@@ -2819,6 +2819,12 @@ of :class:`enum.Enum`.
 
         A stream invite that targets an embedded application.
 
+    .. attribute:: role_subscription_purchase
```

Comment:
According to the linked Discord API Docs PR, the attribute is named `ROLE_SUBSCRIPTIONS_PURCHASE`, not `ROLE_SUBSCRIPTION_PURCHASE`.